src/components/DropMenu/DropMenu.tsx

```diff
@@ -552,8 +554,15 @@ class DropMenu extends React.Component<IDropMenuProps, IDropMenuState> {
 	};
 
 	static preprocessOptionData = (
-		props: IDropMenuProps,
-		ParentType = DropMenu
+		props: StandardProps,
+		//TODO: ask about this fix
+		ParentType: {
```

Comment:
This needed to support passing components other than DropMenu as ParentType. After taking a look at it, I thought `typeof` seemed a bit cleaner than using generic types since it would require 4 type arguments.

Reply:
whether we keep this type or create a new one we should remove the TODO comment

Reply:
thx. removed.